### PR TITLE
feat(tekton): add CVE summary and signed badge

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
**Fixes**: 
#986 
#987 

**Description**: 
Adds a CVE summary column to the Pipelinerun list view
Adds a signed badge indicator to Pipelineruns in the list view that have the `signed` annotation set. 

**Screenshot**
image

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
cc: @ShiranHi @rohitkrai03

File: plugins/shared-react/src/types/pipeline/pipelineRun.ts [hunk 1]
```diff
@@ -87,7 +87,8 @@ export type PipelineRunStatus = {
   skippedTasks?: {
     name: string;
   }[];
-  pipelineResults?: TektonResultsRun[];
+  pipelineResults?: TektonResultsRun[]; // in tekton v1 pipelineResults is renamed to results
```
Comment (gashcrumb) [inline plugins/shared-react/src/types/pipeline/pipelineRun.ts:90]: Based on this comment, is this a good time to rename the type so it's aligned with v1?

Comment (jeff-phillips-18) [reply plugins/shared-react/src/types/pipeline/pipelineRun.ts:90]: We are trying to support both so I've left both here and check for either when looking for results.

**Comment by karthikjeeyar** [comment]
need a custom sorting logic for vulnerabilities column.

**Comment by karthikjeeyar** [comment]
You will be adding a custom sorting logic for vulnerabilities column, right?

File: plugins/tekton/src/components/PipelineRunList/PipelineRunRow.tsx [hunk 2]
```diff
@@ -46,12 +58,28 @@ type PipelineRunRowProps = {
...
     <div>
       {name ? (
-        <ResourceBadge color={tektonGroupColor} abbr="PLR" name={name || ''} />
+        <ResourceBadge
+          color={tektonGroupColor}
+          abbr="PLR"
+          name={name || ''}
+          suffix={
+            signed ? (
+              <Tooltip title="Signed">
```
Comment (karthikjeeyar) [inline plugins/tekton/src/components/PipelineRunList/PipelineRunRow.tsx:75]: Action button uses PF tooltip, so can we use PF tooltip instead material ui here as well?
image

**Comment by ShiranHi** [comment]
Looks good to me, can we add a tooltip when hovering the icons?

**Comment by invincibleJai** [comment]
@jeff-phillips-18 Thanks Jeff for your contribution, how can I test with CVEs? is there a step I can follow?

**Comment by jeff-phillips-18** [comment]
> @jeff-phillips-18 Thanks Jeff for your contribution, how can I test with CVEs? is there a step I can follow?

@invincibleJai You can add this task: https://gist.github.com/karthikjeeyar/d9fe4d12fbbc1d01273c8540b5504d87

File: plugins/tekton/src/components/PipelineRunList/PipelineRunVulnerabilities.tsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+type PipelineRunVulnerabilitiesProps = {
+  pipelineRun: PipelineRunKind;
+  condensed?: boolean;
+};
+
+const PipelineRunVulnerabilities: React.FC<PipelineRunVulnerabilitiesProps> = ({
+  pipelineRun,
+  condensed,
+  // classes,
```
Comment (karthikjeeyar) [inline plugins/tekton/src/components/PipelineRunList/PipelineRunVulnerabilities.tsx:63]: nit: 
```suggestion

```

**Review by karthikjeeyar — APPROVED** [review]
Tested locally, works as expected.

**Review by invincibleJai — APPROVED** [review]
LGTM

**Pull request merged**